.eslintrc

```diff
@@ -76,6 +77,9 @@
     "jest/valid-expect": "error",
     "react/button-has-type": 0,
     "jsx-a11y/label-has-for": 0,
+    "jsx-a11y/click-events-have-key-events": 0,
```

Comment:
I'm not convinced we should disable all of these. We should try to be accessible as possible at a min in this library. We already do a poor job in most of our apps so it would be good to set the precedent in react-common.

Reply:
In general i agree, but this is a company-wide plan that should be mostly driven by UX and product folks, because it requires some specific keyboard behaviours to be introduced. I'd glad if such thing would become real, but afaik we had some approaches for it and we failed in execution. 

Having that rule for the sake of adding empty `onKeyPress` listeners to satisfy linter did not make sense, so I've decided to disable it after 5th or 6th `eslint disable next line` comment.